docs/pr_review_process.md

```diff
@@ -0,0 +1,82 @@
+2. **Process**: in using the code, you think the process can be simplified or made clearer (e.g., fewer inputs, renamed arguments, etc.).
+3. **Documentation**: both the documentation and docstrings should match the code in terms of syntax and process.
+
+Iterate as needed.
+
+### Step 4: Review the tests
+
+1. Do all tests pass?
+    1. Running relevant tests locally: `pytest -v tests/tests.py -k test_name_or_keywords` in a local terminal with an activated PlantCV environment
+2. Does coverage remain at 100%?
+3. Do any new or modified tests accurately test the PR code?
+
+Propose updates as needed.
+
+### Step 5: Test the documentation
+
+Reviewing the documentation code is important, but compiling the documentation locally ensures that the resulting pages are rendered correctly. The easiest way to view the documentation live on your local machine is to run: `mkdocs serve --theme readthedocs` in a local terminal with an activated PlantCV environment
+
+Some common oversights that are hard to see in the code but easy to detect in the compiled documentation:
+
+1. New pages are not added to the table of contents plantcv/mkdocs.yml (mkdocs will display a warning when you run the command above).
+2. Typos in page or image filenames (mkdocs will often display a warning, or the image may appear as a broken link).
+3. Typos in formatting markup lead to incorrectly displayed styling (e.g., not closing bold or italics, not having a blank line before a bulleted or numbered list, etc.).
+4. Changes to function input/output signatures or new functions should be added to updating.md.
+5. Link out to source code at the end of a documentation page.
+
+### Step 6: Review the code
+
+Once the PR is working as intended, review the code itself for potential improvements. Some common types of improvements to suggest:
+
+1. **Uniform style (linting)**: Most IDEs, including PyCharm and VScode, will indicate when the code does not match the community style guides laid out in PEP8. The command-line tool `flake8` can also be used if an IDE that supports these checks cannot be used.
+2. **Code simplification**: Any reorganization that makes the code more succinct, readable, maintainable, etc. In particular, complex, nested logical blocks and for loops increase code complexity (and testing demand).
+3. **Algorithmic improvement**: Could potentially be an alternate approach that is significantly faster or more user-friendly.
+4. **Scope**: Check any new code and test data in the `plantcv/tests` module (no need to fix out-of-scope linting issues).
+5. **Documentation**: The documentation is more flexible, but try to avoid very long lines. Markdown will automatically connect consecutive lines that do not have a blank line between them. Crop down images in documentation. In general, resizing such that the image width is equal to 400 will result in a documentation page that is still mobile friendly.
```

Comment:
```suggestion
5. **Documentation**: The documentation is more flexible, but try to avoid very long lines. Markdown will automatically connect consecutive lines that do not have a blank line between them. Crop down images in documentation. In general, resizing such that the image width is equal to 400px will result in a documentation page that is still mobile friendly.
```